### ISSUE TITLE
[HOLD for payment 2023-10-23]

### ISSUE THREAD

**Issue by aldo-expensify** [Expensify/App#27691]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to DM chat
2. Click + and then "Request Money"
3. Use "Scan" option and try to take a picture

## Expected Result:
After giving permissions to the app, the camera should appear

## Actual Result:
After giving permissions to the app, a loading spinner stays there forever

![image](https://github.com/Expensify/App/assets/87341702/cdfad0e3-48a5-4845-bc2b-48a0ebb50675)

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native - Specifically a Xiaomi A3 device
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:**
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
**Expensify/Expensify Issue URL:**
**Issue reported by:**
**Slack conversation:**
- https://expensify.slack.com/archives/C057F53HR40/p1693966434834629?thread_ts=1693598703.818309&cid=C057F53HR40
- https://expensify.slack.com/archives/C035J5C9FAP/p1693955621069789

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01d479b9dfa637bcca</li>
        <li>Upwork Job ID: 1704032011382767616</li>
        <li>Last Price Increase: 2023-09-19</li>
    </ul>
</details>

**Comment by aldo-expensify** [comment]
Solution:

Marc fixed this here: https://github.com/mrousavy/react-native-vision-camera/pull/1782

We need to update `react-native-vision-camera` to get the fix

**Comment by MitchExpensify** [comment]
Is the idea to label this external still @aldo-expensify ? I do not have access to a Xiaomi so I'm tempted to go straight to that step 😅

**Comment by AndrewGable** [comment]
This was released in `2.16.1` - I can update it

**Comment by MitchExpensify** [comment]
Reminder set to pay! Bump on BZ steps @ntdiary

**Comment by MitchExpensify** [comment]
So we should hold on payment if this is still happening, right @ntdiary ?

**Comment by lanitochka17** [comment]
Issue still reproducible with latest build 1.3.99-0
Samsung Galaxy Z Fold 4 / Android 13

https://github.com/Expensify/App/assets/78819774/d5b7429a-8ad4-47c5-b4f7-db6c0cc4596b

**Comment by AndrewGable** [comment]
What's the latest here?

**Comment by MitchExpensify** [comment]
I am unsure honestly, @AndrewGable - We need a proposal to fix it because our original fix here didn't do that, right?

**Comment by MitchExpensify** [comment]
Friendly bump, @AndrewGable

**Comment by MitchExpensify** [comment]
And just to be clear we'd wait on https://github.com/Expensify/App/issues/30732. in the hope that it resolves this issue?

**Comment by AndrewGable** [comment]
Yes, correct

**Comment by MitchExpensify** [comment]
On hold, not overdue

**Comment by MitchExpensify** [comment]
On hold for https://github.com/Expensify/App/issues/30732

**Comment by MitchExpensify** [comment]
Hold for https://github.com/Expensify/App/issues/30732 which found a home on a wave so progress will continue

**Comment by MitchExpensify** [comment]
Hold for https://github.com/Expensify/App/issues/30732

**Comment by MitchExpensify** [comment]
Hold for https://github.com/Expensify/App/issues/30732

**Comment by MitchExpensify** [comment]
Hold for https://github.com/Expensify/App/issues/30732

**Comment by MitchExpensify** [comment]
Hold for https://github.com/Expensify/App/issues/30732

**Comment by MitchExpensify** [comment]
Hold for https://github.com/Expensify/App/issues/30732

**Comment by MitchExpensify** [comment]
Hold for https://github.com/Expensify/App/issues/30732
